hosting_providers/index.md

```diff
@@ -26,8 +26,14 @@ If you’re not using the shared hosting product, or if your provider doesn’t
 
 **_The list is categorized into:_**
 
-<a id="full-bounce" href="#table-anchor">Full HTTPS Support</a>: Hosting providers that offer free Let’s Encrypt HTTPS certificates automatically for their shared hosting product. No need to do anything on your end if you use their service.
-
-<a id="partial-bounce" href="#table-anchor">Partial HTTPS Support</a>: Hosting providers that offer free Let’s Encrypt HTTPS certificates for their shared hosting product, but require configuration. You’ll need to follow a tutorial.
-
-<a id="no-bounce" href="#table-anchor">No HTTPS Support</a>: Hosting providers that do not offer free HTTPS certificates for their shared hosting product. It may or may not be possible to use Certbot with other products that the provider offers, such as their VPS product.
+<div class="provider-categories">
+  <div>
+    <a id="full-bounce" href="#table-anchor"><img src="/images/GreenCheck.svg" alt="green checkmark icon">Full HTTPS Support</a>: Hosting providers that offer free Let’s Encrypt HTTPS certificates automatically for their shared hosting product. No need to do anything on your end if you use their service.
+    </div>
```

Comment:
Can you clean up the whitespace around here? It's hard to tell at a glance how these divs are nested when the closing tags don't align.

Reply:
Yep.